src/index.html

```diff
@@ -7,12 +7,68 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+      href="https://fonts.googleapis.
+      com/css2?family=Poppins:ital,wght@0,100;
+      0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;
+      1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;
+      1,900&family=Roboto:ital,wght@0,100;0,300;0,400;
+      0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;
+      1,900&display=swap" rel="stylesheet">
```

Comment:
There is a typo in the URL for the Google Fonts stylesheet. The URL is split across multiple lines, which might cause issues. Ensure the URL is complete and correctly formatted on a single line to avoid loading errors.